docs/dev/libvirt-howto.md

```diff
@@ -223,14 +223,7 @@ Set `TAGS` when building if you need `destroy cluster` support for libvirt; this
 TAGS=libvirt_destroy hack/build.sh
 ```
 
-To avoid being prompted repeatedly, you can set [environment variables](../user/environment-variables.md) to reflect your libvirt choices.  For example, selecting libvirt, setting [our earlier name choices](#pick-names), and telling both the installer and the machine-API operator to contact `libvirtd` at [the usual libvirt IP](#firewall), you can use:
-
-```sh
-export OPENSHIFT_INSTALL_PLATFORM=libvirt
-export OPENSHIFT_INSTALL_BASE_DOMAIN=tt.testing
-export OPENSHIFT_INSTALL_CLUSTER_NAME=test1
-export OPENSHIFT_INSTALL_LIBVIRT_URI=qemu+tcp://192.168.122.1/system
-```
+To avoid being prompted repeatedly, you can create an install config one time and then provide that to future installations.
```

Comment:
This advice is no longer libvirt-specific.  Can we drop it entirely and add some generic docs (to the README's quick-start?) for this workflow?

Reply:
I had started on a general customization doc. I can add it to that, but it's not ready to go yet. What do you think about merging this now and following up with a docs PR?

Reply:
Scratch that. The customization doc isn't the best place for that note. I'm going to start a tips and tricks doc.